app/styles/stepper.module.css

```diff
@@ -27,14 +27,15 @@
 }
 
 .label {
-  color: var(--color-black);
-  font-weight: 700;
-  font-size: 1.25rem;
-  margin: 0 0 2px 2px;
+  color: var(--neutral-neutral-800, #1F1F1F);
+  font-size: 1rem;
+  font-style: normal;
+  font-weight: 550;
```

Comment:
How 550 will work 
Don't chnage previous value until new design is not  there

Reply:
Done with changes Can you check my latest commit